woocommerce/woocommerce-android
# [Better Tablet Products] Update visibility of the menu items on the Variations screen when the state changes

Closes: #11123 


### Description

As we use individual toolbar here, `invalidateOptionsMenu` didn't do much and the visibility of the menu items were not changed on the state changes resulting in missing items in 2 pane mode

### Testing instructions

Test both in 2 and 1 panes mode
* Prodcuts -> Product -> Variations 
* Notice that "bulk update" menu item is there

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/c67451f9-7e7b-49b6-ae7a-18fdb2241903
